gateway-messages/src/sp_to_mgs.rs

```diff
@@ -163,6 +166,22 @@ pub struct SpState {
     pub rot: Result<RotState, RotError>,
 }
```

Comment:
Maybe add a comment on `SpState` that it is quasi-deprecated and will only be returned by SPs with images older than the introduction of `SpStateV2`?

Reply:
Fixed in [8d4cea8](https://github.com/oxidecomputer/management-gateway-service/pull/92/commits/8d4cea8b6861908414ee76f068cf212622eda10d)